centinel/backend.py

```diff
@@ -29,7 +30,8 @@ def request(self, slug):
         url = "%s/%s" % (self.config['server']['server_url'], slug)
         try:
             req = requests.get(url, auth=self.auth,
-                               proxies=self.config['proxy']['proxy'])
+                               proxies=self.config['proxy']['proxy'],
+                               verify=self.verify)
```

Comment:
When did we drop the verification and why are we adding it back now? Also, why use the full config name for the proxies part but not for verify?